### PR TITLE
UC plugin packages for GI

### PR DESCRIPTION
**Pull request by RefaelAdi**
Added the following packages for GI:
couchbaseDB
mysql on AWS
mysql on prem
postgres on AWS

and I updated the available_plugins.md file

### CODE REVIEW

**Review by natalygmkibm — APPROVED** [review]

File: filter-plugin/logstash-filter-couchbasedb-guardium/CouchbasedbOverFilebeatPackage/gi_templates.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-couchbasedb-guardium/CouchbasedbOverFilebeatPackage/gi_templates.json:1]: We should try to create a manual for this file

**Review by ofer-haim — APPROVED** [review]

**Review by ofer-haim — APPROVED** [review]
Please resolve the conflicts

File: filter-plugin/logstash-filter-mysql-aws-guardium/MysqlOverCloudwatchLogsPackage/MySQL/filter.conf [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    add_field => {
+      "[GuardRecord][sessionId]" => "%{[cloudwatch_logs][event_id]}"
+      "[GuardRecord][accessor][dbProtocol]" => "MySQL native audit"
+      "[GuardRecord][accessor][dataType]" => "TEXT"
+      "[GuardRecord][accessor][language]" => "MYSQL"
+      "[GuardRecord][accessor][serverType]" => "MySql"
+      "[GuardRecord][accessor][serverOS]" => ""
+      "[GuardRecord][accessor][clientOs]" => ""
+      "[GuardRecord][accessor][clientHostName]" => ""
+      "[GuardRecord][accessor][serverHostName]" => "rds.mysql.amazonaws.com"
```
[the remark this answers is not in the document]
Comment (RefaelAdi) [reply filter-plugin/logstash-filter-mysql-aws-guardium/MysqlOverCloudwatchLogsPackage/MySQL/filter.conf:60]: Thanks, Tal.
I have an open [issue](https://jira.rtp.raleigh.ibm.com:8443/browse/GRD-58977) to implement DB name changes with account id (following PostgreSQL plugin, the serverHostName consists of the account id and instance).
I'll add it to that open issue.

File: filter-plugin/logstash-filter-couchbasedb-guardium/CouchbasedbOverFilebeatPackage/gi_templates.json (hunk 1 above)
Comment (RefaelAdi) [reply filter-plugin/logstash-filter-couchbasedb-guardium/CouchbasedbOverFilebeatPackage/gi_templates.json:1]: Ok, thanks.

File: docs/available_plugins.md [hunk 3]
```diff
@@ -6,18 +6,18 @@ Pre-packaged plugins can be downloaded from here: https://github.com/IBM/univers
 | ------------------------------------------------------------------------- | --------------- | ------------------------ | -------------------- | ------------------------------ | ----------------------------- |
 | [Amazon S3](../filter-plugin/logstash-filter-s3-guardium/README.md)          |                 | AWS                      | IBM                  | CloudWatch (pull), SQS (pull)  |                               |
 | [HDFS](../filter-plugin/logstash-filter-hdfs-guardium/README.md)          | Hadoop 3.1.x    | On-premise / Iaas        | IBM                  | Filebeat (push)                | **Guardium Data Protection and GI** |
-| [MySQL](../filter-plugin/logstash-filter-mysql-guardium/README.md)        | 5.x             | On-premise / Iaas        | IBM                  | Syslog (push), Filebeat (push) |                               |
+| [MySQL](../filter-plugin/logstash-filter-mysql-guardium/README.md)        | 5.x             | On-premise / Iaas        | IBM                  | Syslog (push), Filebeat (push) | **Guardium Data Protection and GI** |
```
[the remark this answers is not in the document]
Comment (RefaelAdi) [reply docs/available_plugins.md]: removed

[the remark this answers is not in the document]
Comment (ofer-haim) [reply docs/available_plugins.md]: Refael, this needs to be removed from *all* of the rows you added it, not only this one

**Pull request merged**